content/lxd/getting-started-cli.md

```diff
@@ -164,7 +164,13 @@ Run the following command to start the interactive configuration process:
 
 **Quick minimal setup**
 
-To create a non-optimized minimal setup with default options, you can skip the configuration steps by adding the `--auto` flag:
+To create a non-optimized minimal setup with default options, you can skip the configuration steps by adding the following flags:
+
+For LXD 4.18 and newer, use:
+
+    sudo lxd init --minimal
```

Comment:
```suggestion
To create a non-optimized minimal setup with default options, you can skip the configuration steps by adding the ``--minimal`` flag (``--auto`` in LXD versions prior to 4.18):

    sudo lxd init --minimal
```

Reply:
1. I think that the guide should probably cover both the feature release and 4.0.x lts variant. 
2. This is only an intermediant solution, when 4.0.8 is released, we can simply just use `--minimal`.
   So right now I think my version is better readable.

Reply:
I'd probably just wait for 4.0.8 to be out and then make the change.

Reply:
Ok, then I will wait for that.